Makefile

```diff
@@ -1,69 +1,70 @@
+    export HOME := $(shell cd ~ && pwd)
+endif
 
 ifeq ($(OS_NAME),Linux)
-	RUNTIME_ID := $(shell . /etc/os-release && echo $$ID.$$VERSION_ID)-x64
+    RUNTIME_ID := $(shell . /etc/os-release && echo $$ID.$$VERSION_ID)-x64
 else ifeq ($(OS_NAME),Darwin)
-	RUNTIME_ID := osx.10.12-x64
+    RUNTIME_ID := osx.10.12-x64
+else
+    $(error "Unknown OS_NAME: $(OS_NAME)")
 endif
 
+MSBUILD_ARGS := /v:m /fl /fileloggerparameters:Verbosity=normal /p:Configuration=$(BUILD_CONFIGURATION)
+
 ifneq ($(BUILD_LOG_PATH),)
-	MSBUILD_ADDITIONALARGS := $(MSBUILD_ADDITIONALARGS) /fileloggerparameters:LogFile=$(BUILD_LOG_PATH)
+    MSBUILD_ARGS := $(MSBUILD_ARGS) /fileloggerparameters:LogFile=$(BUILD_LOG_PATH)
 endif
 
 ifeq ($(BOOTSTRAP),true)
-	MSBUILD_ARGS = $(MSBUILD_ADDITIONALARGS) /p:CscToolPath=$(BOOTSTRAP_PATH)/csc /p:CscToolExe=csc /p:VbcToolPath=$(BOOTSTRAP_PATH)/vbc /p:VbcToolExe=vbc
-else
-	MSBUILD_ARGS = $(MSBUILD_ADDITIONALARGS)
+    MSBUILD_ARGS := $(MSBUILD_ARGS) /p:CscToolPath=$(BOOTSTRAP_PATH)/csc /p:CscToolExe=csc /p:VbcToolPath=$(BOOTSTRAP_PATH)/vbc /p:VbcToolExe=vbc
 endif
 
-BUILD_CMD = dotnet build $(MSBUILD_ARGS)
+BUILD_CMD := dotnet build $(MSBUILD_ARGS)
 
 .PHONY: all bootstrap test toolset
 
 all: restore
-	@export PATH="$(BINARIES_PATH)/dotnet-cli:$(PATH)" ; \
-	export HOME="$(HOME_DIR)" ; \
-	$(BUILD_CMD) CrossPlatform.sln
+	$(BUILD_CMD) $(THIS_MAKEFILE_PATH)/CrossPlatform.sln
 
 bootstrap: restore
-	export HOME="$(HOME_DIR)" ; \
-	export PATH="$(BINARIES_PATH)/dotnet-cli:$(PATH)" ; \
-	$(BUILD_CMD) src/Compilers/CSharp/CscCore && \
-	$(BUILD_CMD) src/Compilers/VisualBasic/VbcCore && \
-	mkdir -p $(BOOTSTRAP_PATH)/csc && mkdir -p $(BOOTSTRAP_PATH)/vbc && \
-	dotnet publish -c $(BUILD_CONFIGURATION) -r $(RUNTIME_ID) src/Compilers/CSharp/CscCore -o $(BOOTSTRAP_PATH)/csc && \
-	dotnet publish -c $(BUILD_CONFIGURATION) -r $(RUNTIME_ID) src/Compilers/VisualBasic/VbcCore -o $(BOOTSTRAP_PATH)/vbc
-	rm -rf Binaries/$(BUILD_CONFIGURATION)
+	$(BUILD_CMD) $(SRC_PATH)/Compilers/CSharp/CscCore
+	$(BUILD_CMD) $(SRC_PATH)/Compilers/VisualBasic/VbcCore
+	mkdir -p $(BOOTSTRAP_PATH)/csc
+	mkdir -p $(BOOTSTRAP_PATH)/vbc
+	dotnet publish -c $(BUILD_CONFIGURATION) -r $(RUNTIME_ID) $(SRC_PATH)/Compilers/CSharp/CscCore -o $(BOOTSTRAP_PATH)/csc
+	dotnet publish -c $(BUILD_CONFIGURATION) -r $(RUNTIME_ID) $(SRC_PATH)/Compilers/VisualBasic/VbcCore -o $(BOOTSTRAP_PATH)/vbc
+	rm -rf $(BINARIES_PATH)/$(BUILD_CONFIGURATION)
 
 test:
-	@export PATH="$(BINARIES_PATH)/dotnet-cli:$(PATH)" ; \
-	export HOME="$(HOME_DIR)" ; \
-	dotnet publish -r $(RUNTIME_ID) src/Test/DeployCoreClrTestRuntime -o $(BINARIES_PATH)/$(BUILD_CONFIGURATION)/CoreClrTest -p:RoslynRuntimeIdentifier=$(RUNTIME_ID) && \
-	build/scripts/tests.sh $(BUILD_CONFIGURATION)
+	dotnet publish -r $(RUNTIME_ID) $(SRC_PATH)/Test/DeployCoreClrTestRuntime -o $(BINARIES_PATH)/$(BUILD_CONFIGURATION)/CoreClrTest -p:RoslynRuntimeIdentifier=$(RUNTIME_ID)
+	./build/scripts/tests.sh $(BUILD_CONFIGURATION)
```

Comment:
No difference. Changed it for consistency: nearby we invoke `./build/scripts/restore.sh`. I'll update both to use `$(THIS_MAKEFILE_PATH)` (which apparently ends with a slash, didn't read the Makefile docs for `dir` well enough, I'll update that too), and any other relative-to-pwd things I find.